### PR TITLE
Add October 2023 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
----
:books: Documentation preview :books:: https://writethedocs-www--2037.org.readthedocs.build/

### CODE REVIEW

File: docs/blog/newsletter-october-2023.rst
Comment (plaindocs) [inline docs/blog/newsletter-october-2023.rst]: Speakers should be announced soon, right @swapnilogale ?

File: docs/blog/newsletter-october-2023.rst
Comment (plaindocs) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
There are still different views among documentarians on the issue of the importance of personality, customization, and tone in technical writing.
```

harsh, but...

File: docs/blog/newsletter-october-2023.rst
Comment (plaindocs) [inline docs/blog/newsletter-october-2023.rst]: Random passive for no reason here?

File: docs/organizer-guide/newsletter/newsletter-process.rst
Comment (plaindocs) [inline docs/organizer-guide/newsletter/newsletter-process.rst:132]: Personal preference for the lower case here, but :man_shrugging:

**Review by plaindocs — COMMENTED** [review]
Looks fine to me.

File: docs/blog/newsletter-october-2023.rst
Comment (swapnilogale) [reply docs/blog/newsletter-october-2023.rst]: Yes, just overnight if I can

File: docs/blog/newsletter-october-2023.rst
Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]: I think the idea was to have variety in sentence intros, but can change it.

Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]:
```suggestion
People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only runing when needed (not linting untouched code).
```

File: docs/organizer-guide/newsletter/newsletter-process.rst
Comment (CollierCZ) [reply docs/organizer-guide/newsletter/newsletter-process.rst:132]: Yeah, I get that. Mostly I just noticed a mixture creeping in and wanted to get away from that and picking either standard seemed better. It's also something of a 🇺🇸 vs. 🇬🇧 thing. 🤷

Comment (plaindocs) [reply docs/organizer-guide/newsletter/newsletter-process.rst:132]: For sure your call, and for sure consistency is better than a mix.

File: docs/blog/newsletter-october-2023.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2023.rst]: "the most important in technical writing" seems like it needs a noun, like "the most important consideration in technical writing"

**Comment by swapnilogale** [comment]
@CollierCZ any chance of holding this newsletter from going out until Monday? Just hoping to get the speakers announced in a day or two

File: docs/blog/newsletter-october-2023.rst
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only running when needed (not linting untouched code).
```

File: docs/blog/newsletter-october-2023.rst
Comment (plaindocs) [reply docs/blog/newsletter-october-2023.rst]: @hillaryfraley  Good call! Alternatively "accuracy was most important in ..."

**Review by hillaryfraley — APPROVED** [review]

File: docs/blog/newsletter-october-2023.rst
Comment (plaindocs) [reply docs/blog/newsletter-october-2023.rst]: Sad to see no rune emoji

**Comment by CollierCZ** [comment]
> @CollierCZ any chance of holding this newsletter from going out until Monday? Just hoping to get the speakers announced in a day or two

Sure thing.

File: docs/blog/newsletter-october-2023.rst
Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]: I remember changing it to that and thinking, "Remember to come back and add a noun." 🤦 Thanks!

Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]:
```suggestion
Some people said that strict accuracy was the most important consideration in technical writing, while others said that there was room for creativity and humor as long as precision wasn’t compromised. In particular, they said that adding a bit of personality can make such writing more interesting, especially if the readers can handle it.
```

File: docs/blog/newsletter-october-2023.rst
Comment (CollierCZ) [reply docs/blog/newsletter-october-2023.rst]: That's because it's already a part of Unicode (ᚠ,ᛃ,ᚬ): https://en.wikipedia.org/wiki/Runic_(Unicode_block)

**Comment by plaindocs** [comment]
@Aaron228 I've just merged the Australian speaker announcement, so as long as you can see the news post and list of speakers, it's good to go.

File: docs/blog/newsletter-october-2023.rst
Comment (CollierCZ) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```

File: docs/blog/newsletter-october-2023.rst
Comment (CollierCZ) [inline docs/blog/newsletter-october-2023.rst]:
```suggestion
In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>`__. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```

**Pull request merged**
